DataFormats/PortableTestObjects/src/alpaka/classes_serial.h

```diff
@@ -2,3 +2,14 @@
 #include "DataFormats/Portable/interface/Product.h"
 #include "DataFormats/PortableTestObjects/interface/TestHostCollection.h"
 #include "DataFormats/PortableTestObjects/interface/TestSoA.h"
+#include "DataFormats/Portable/interface/PortableCollectionCommon.h"
+
+namespace ROOT::Meta::Selection {
+  namespace portablecollection {
+    class c0 : CollectionImpl<0, void> {};
+
+    class CL0_TestHostCollection : CollectionLeaf<0, portabletest::TestSoALayout<128, false>> {};
```

Comment:
`CL0_TestHostMultiCollection` ?

Reply:
It belongs to all 3 collections (single and multi). I will put `Multi` everywhere for uniformity.